src/components/SearchResults.tsx

```diff
@@ -8,22 +10,33 @@ type SearchResultsProps = {
     priceFormatted: string;
     title: string;
   }>;
-  onAddToWishlist: (id: number) => void;
+  onAddToWishlist: (id: number) =>   void;
 }
 
 export function SearchResults({ results, onAddToWishlist, totalPrice }: SearchResultsProps) {
+  const rowRenderer: ListRowRenderer = ({ index, key, style }) => {
+    return (
+      <div key={key} style={style}>
+        <ProductItem
+          product={results[index]}
+          onAddToWishlist={onAddToWishlist}
+        />
+      </div>
+    );
+  }
```

Comment:
Render function that returns a JSX and display virtualized component for each child.